js/scripts.js

```diff
@@ -4,7 +4,7 @@ var maleNames = ["Piotrek", "Marek", "Arek", "Jarek"];
 var allNames = femaleNames.concat(maleNames);
 var newName = "Marian";
 
-if (allNames.indexOf(newName) === -1) 
```

Comment:
nie musisz robić `if(allNames.includes(...) === false)`, wystarczy zrobienie `if (!allNames.includes(...))`

jeżeli wewnątrz `if` będzie wartość tzw `truthy` to zostanie wykonany kod wewnątrz.

https://www.sitepoint.com/javascript-truthy-falsy/